src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/DraftReferralService.kt

```diff
@@ -108,6 +114,7 @@ class DraftReferralService(
     updateServiceUserNeeds(referral, update)
     updateDraftRiskInformation(referral, update)
     updateServiceCategoryDetails(referral, update)
+    updatePersonCurrentLocation(referral, update)
```

Comment:
There is a feature flag around the validate functionality but not this, meaning that it will just be stored without any validation at all. Should this also be inside the feature flag?